tools/cldr/build.xml

```diff
@@ -0,0 +1,71 @@
+<!-- © 2020 and later: Unicode, Inc. and others.
+     License & terms of use: http://www.unicode.org/copyright.html -->
+
+<!-- FIXME: more instructions here. -->
```

Comment:
Cross-references may be useful, as a minimum. Perhaps just point at icu4c/source/data/cldr-icu-readme.txt from here?

Reply:
Sounds good.